reference-artifacts/SCPs/ASEA-Guardrails-Sandbox.json

```diff
@@ -47,11 +47,11 @@
       "Resource": "*",
       "Condition": {
         "StringNotEquals": {
-          "aws:RequestedRegion": ["ca-central-1", "us-east-1", "us-east-2", "us-west-1", "us-west-2"]
+          "aws:RequestedRegion": ["${HOME_REGION}", "${GBL_REGION}", "us-east-2", "us-west-1", "us-west-2"]
```

Comment:
How are we replacing all the regions as one variable?  in theory this should be:
`"aws:RequestedRegion": ["${ADDL_REGION_C}"]`
but I don't know how you are handling the quotes required - does the replacements section need to change from an array to a string???

Reply:
This should be as follows, "aws:RequestedRegion": "${ADDL_REGION_C}"

We decide replace with quotes or not based on replacement value. In this case since "ADDL_REGION_C" is a list replacement we will replace including "". In case of sting we just replace ${repl_str} with value.